src/Controls/src/Core/ObservableWrapper.cs

```diff
@@ -291,10 +298,7 @@ public void Remove(object value)
 			Remove((TRestrict)value);
 		}
 
-		public void CopyTo(Array array, int index)
-		{
-			CopyTo(array.Cast<TRestrict>().ToArray(), index);
```

Comment:
This code didn't work before, right? Wasn't this copying to a new array which then just got discarded?

Reply:
Same here, I think I can write a test that breaks.